Components/Toasts/toast.css

```diff
@@ -0,0 +1,48 @@
+.toast{
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    /* width: 100%; */
+    max-width: 500px;
+    margin: 0px 1rem 0px 1rem;
+    font-size: 1rem; 
```

Comment:
If you use margin: 0px 1rem; , it will give same effect. When top-bottom and left-right margin are same, you can use this shorthand.

Reply:
Ohh yes, thanks for noticing the difference😉